src/components/radio-button/radio-button/radio-button.scss

```diff
@@ -147,7 +147,7 @@ slot[name='subtext'] {
     margin-block-start: var(--sbb-radio-button-icon-align);
 
     transition: {
-      duration: var(--sbb-animation-duration-4x);
+      duration: var(--sbb-disable-animation-time, var(--sbb-animation-duration-4x));
```

Comment:
should we also document this kind of components where you newly introduced the disable animation logic? (in the corresponding readme.md files)

Reply:
Probably yes. Or even better, I'd like to completely remove it from readmes in favor of the "getting started" doc. Let's also ask @kyubisation  what does he think

Reply:
Since it is a global topic, I would prefer a section in getting started (or similar).
But I'm open to discuss this.

Reply:
I will remove readme references in favor of the getting started doc, anyone opposing?

Reply:
for me it's ok. maybe you can check the corresponding section in my open PR, and also, maybe, extend yourself there with a commit? ;)